When : everytime
Why : Fix the run dir to comply with the Gentoo FS layout.
--- a/configure.ac
+++ b/configure.ac
@@ -337,7 +337,7 @@
 
 manager_failover_dir=$LOCALSTATEDIR/spool/prelude-manager/failover
 manager_scheduler_dir=$LOCALSTATEDIR/spool/prelude-manager/scheduler
-manager_run_dir=$LOCALSTATEDIR/run/prelude-manager
+manager_run_dir=/run/prelude-manager
 
 AC_DEFINE_UNQUOTED(REPORT_PLUGIN_DIR, "$report_plugin_dir", Prelude-Manager report plugin directory)
 AC_DEFINE_UNQUOTED(DECODE_PLUGIN_DIR, "$decode_plugin_dir", Prelude-Manager decode plugin directory)
